### DISCUSSION TITLE
Is filament table support with spatie query builder

### DISCUSSION THREAD

**Discussion by zulfikar-ditya** [filamentphp/filament#7765] [Help]
is filament table support with [spatie query builder](https://spatie.be/docs/laravel-query-builder/v5/introduction)

**Comment by pboivin** [comment]
What feature do you need from this package that is not in Filament?

**Reply by zulfikar-ditya** [reply]
i am just asking, i use datatable for web and api, i using repository pattern, so is posible to use spatie query builder in filament, if not support  i just need create one function in my repository
